verify/kds/kds.go

```diff
@@ -361,3 +364,27 @@ func ParsePlatformCertChain(pems []byte) ([]byte, []byte, error) {
 	}
 	return askBlock.Bytes, arkBlock.Bytes, nil
 }
+
+func platformBaseURL(name string) string {
```

Comment:
Comment to describe the method?

Reply:
Done, but the style guide doesn't require comments on unexported functions.